WordPress/src/main/java/org/wordpress/android/ui/domains/management/Theme.kt

```diff
@@ -0,0 +1,94 @@
+package org.wordpress.android.ui.domains.management
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material3.ColorScheme
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.ProvideTextStyle
+import androidx.compose.material3.Surface
+import androidx.compose.material3.darkColorScheme
+import androidx.compose.material3.lightColorScheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.graphics.Color
+import org.wordpress.android.ui.compose.theme.AppColor
+
+
+data class ExtraColors(
+    val success: Color,
+    val warning: Color,
+)
+
+private val extraPaletteLight = ExtraColors(
+    success = AppColor.JetpackGreen50,
+    warning = AppColor.Orange50,
+)
+
+private val extraPaletteDark = ExtraColors(
+    success = AppColor.JetpackGreen30,
+    warning = AppColor.Orange40,
+)
+
+private val localColors = staticCompositionLocalOf { extraPaletteLight }
+
+@Composable
+fun M3Theme(
+    isDarkTheme: Boolean = isSystemInDarkTheme(),
+    content: @Composable () -> Unit
+) {
+    M3ThemeWithoutBackground(isDarkTheme) {
+        ContentInSurface(content)
+    }
+}
+
+@Composable
+fun M3ThemeWithoutBackground(
+    isDarkTheme: Boolean = isSystemInDarkTheme(),
+    content: @Composable () -> Unit
+) {
+    val extraColors = if (isDarkTheme) {
+        extraPaletteDark
+    } else {
+        extraPaletteLight
+    }
+    CompositionLocalProvider (localColors provides extraColors) {
+        MaterialTheme(
+            colorScheme = if (isDarkTheme) {
+                darkColorScheme(
+                    error = AppColor.Red30,
+                )
+            } else {
+                lightColorScheme(
+                    error = AppColor.Red50,
+                )
+            },
+            content = content
+        )
+    }
```

Comment:
I'm curious, why isn't this theme based on `AppTheme` from `org.wordpress.android.ui.compose.theme`? 

In addition, shouldn't we expand the `JpLightColorPalette` and `JpDarkColorPalette` with the extra colors: success and warning?

Reply:
In the future perspective, we could also extract the Jetpack Compose setup, such as theming to a separate module.

Reply:
Good question. I forgot to mention that I left some info in the commit comments (e.g. this one is explained in [953cdc0](https://github.com/wordpress-mobile/WordPress-Android/pull/19269/commits/953cdc0992334ec6cdef9d70db456f441954186d)):

> This is needed because the existing `AppTheme` does not support material
> 3. For now, this is added as specific theme for these designs, but it is
> written with the possibility of hoisting it later for more general use
> within the app.

Does this make sense to you?

Reply:
Yeah, totally! Thank you for the clarification!